packages/manager/core/app/src/components/Dashboard/DashboardTileLoading.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { TileSection } from '@ovh-ux/manager-themes';
+import { SkeletonText } from '@chakra-ui/react';
+
+export default function DashboardTileLoading({ count = 4 }: any): JSX.Element {
```

Comment:
-any :)

Reply:
This component will be useless. With @zakariasahmane, we discussed on a props that will be added to the `Tile` component that will handle the loading.